src/core/execute.c

```diff
@@ -4909,6 +4909,19 @@ static int exec_child(
                         return r;
         }
 
+	if (context->memory_ksm) {
+#ifdef PR_SET_MEMORY_MERGE
```

Comment:
please add a fallback definition of PR_SET_MEMORY_MERGE to src/basic/missing_prctl.h instead.

Reply:
I'll add PR_SET_MEMORY_MERGE to missing_prctl.h in the next version.